macos/FluentUI/DatePicker/DatePickerView.swift

```diff
@@ -104,7 +104,7 @@ class DatePickerView: NSView {
 	}
 	
 	required init?(coder decoder: NSCoder) {
```

Comment:
would recommend updating these to also have @available(*, unavailable) to truly make this a compile-time break